osu.Game/Localisation/EditorSetupStrings.cs

```diff
@@ -0,0 +1,204 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class EditorSetupStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.EditorSetup";
+
+        /// <summary>
+        /// "beatmap setup"
+        /// </summary>
+        public static LocalisableString BeatmapSetup => new TranslatableString(getKey(@"beatmap_setup"), @"beatmap setup");
+
+        /// <summary>
+        /// "change general settings of your beatmap"
+        /// </summary>
+        public static LocalisableString BeatmapSetupDescription => new TranslatableString(getKey(@"beatmap_setup_description"), @"change general settings of your beatmap");
+
+        /// <summary>
+        /// "Colours"
+        /// </summary>
+        public static LocalisableString ColoursHeader => new TranslatableString(getKey(@"colours_header"), @"Colours");
+
+        /// <summary>
+        /// "Hitcircle / Slider Combos"
+        /// </summary>
+        public static LocalisableString HitcircleSliderCombos => new TranslatableString(getKey(@"hitcircle_slider_combos"), @"Hitcircle / Slider Combos");
```

Comment:
```suggestion
        public static LocalisableString HitcircleSliderCombos => new TranslatableString(getKey(@"hitcircle_slider_combos"), @"Hit circle / Slider Combos");
```

https://osu.ppy.sh/wiki/en/Gameplay/Hit_object/Hit_circle